core/load-git.c

```diff
@@ -771,8 +771,8 @@ static int get_divemode(const char *divemodestring) {
  */
 struct parse_event {
 	const char *name;
-	struct event ev;
 	int has_divemode;
+	struct event ev;
```

Comment:
Not that it makes a difference, but the compiler is wrong here. The variable-length array is not accessed.

Reply:
My compiler (clang) complains that this is only allowed in a GNU extension of the standard to have a variable length element not at the end. So why not reshuffle in a way that the standard is happy?

Reply:
Sorry - I didn't want to imply that we shouldn't shut up the compiler - I was just idly pointing out that this is not a real bug. Note that in C++ mode this would cause a hard error, not a warning [because different struct members need to have different addresses, which is not possible if the variable sized array has 0 size].

BTW, we use numerous GNU extensions, such as these weird case labels with ranges.